components/prism-system_verilog.js

```diff
@@ -0,0 +1,23 @@
+Prism.languages.system_verilog = {
+  'comment': /\/\/.*|\/\*[\w\W]*?\*\//,
+  'string': /"(\\\n|\\?.)*?"/,
+  // support for any kernel function (ex: $display())
+  'property': /(?!\b)\$\w+\b/,
+  // support for user defined constants (ex: `define)
+  'constant': /(?!\b)`\w+\b/,
+  'function': {
+    'pattern': /[a-z0-9_]+\(/i,
+    'inside': {
+      'punctuation': /\(/
+    }
```

Comment:
Here, it would probably be better to use a look-ahead instead of the `inside` (since the parenthesis will be highlighted by the `punctuation` pattern anyway):

```
'function': {
    'pattern': /[a-z0-9_]+(?=\()/i
},
```